docs/it/senior.md

```diff
@@ -0,0 +1,400 @@
+# Lo sviluppatore "senior"
+
+L'industria del software si muove alla velocitá della luce.
+
+Come sviluppatori software siamo costantemente bombardati da notizie di _nuove_ tecnologie, _nuovi_ framework, _nuovi_ linguaggi, _nuovi_ paradigmi, _nuovi_ modelli organizzativi, _nuovi_ modelli di sviluppo, _nuovi_ modelli di deployment, _nuovi_ modelli di testing, _nuovi_ modelli di integrazione, _nuovi_ modelli di delivery, _nuovi_ modelli di monitoraggio, etc.
+
+Una valanga travolgente di innovazione che non da segni di rallentamento.
+
+È questa l'arma a doppio taglio nella mani di uno sviluppatore software. Da una parte abbiamo a che fare con infinite opportunitá di crescita e il prospetto di non annoiarci mai. Dall'altra parte, siamo sempre alla rincorsa della "next big thing" e, per quanto ci sforziamo di stare al passo con i tempi, siamo sempre inesorabilmente un passo indietro.
+
+In questo contesto cosí dinamico, cosa significa essere uno sviluppatore "senior"?
```

Comment:
Correggere accenti acuti: es così, velocità ecc (vale per tutto il capitolo)

Inoltre (un puntiglio) lo metto qui generico per tutto il file:
etc. o ecc. è un'abbreviazione e vuole il punto (tendenzialmente in italiano metto ecc. ma credo che vada bene anche etc., la cosa importante è rimanere costanti)

Reply:
vediamo se riesco a trovare un modo semplice per risolvere tutti questi (benedetta tastiera UK su mac...)

Reply:
Fatto grazie a M$ wordz! 🎉🎉

**Word is my new markdown editor!** 💪